theme/templates/base.html

```diff
@@ -22,26 +22,47 @@
         {% endif %}
     </head>
     <body class="{% if page and page.body_class %}{{ page.body_class }}{% endif %}">
-        <!-- Nav -->
-        <nav class="col-12 primary">
-            <a href="/">
-                <img src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/images/kiwi_logo_h50.png" name="logo">
-            </a>
-
-            <ul class="topnav">
-                <li class="right">
+        <!-- Mobile Nav -->
+        <aside class="mobilenav primary">
+            <button class="close-menu accent-unimportant"><i class="fa fa-times" aria-hidden="true"></i></button>
+            <ul>
+                <li><a class="active accent" href="https://demo.kiwitcms.org">Checkout Demo</a></li>
+                <li><a class="accent" href="{{ SITEURL }}/blog/">Blog</a></li>
+                <li><a href="http://mrsenko.com">Support</a></li>
+                <li><a href="https://github.com/kiwitcms/Kiwi">Community</a></li>
+                <li><a href="http://kiwitcms.readthedocs.io">Documentation</a></li>
+                <li>
                     <form action="http://google.com/search" method="get">
                         <input name="sitesearch" value="http://kiwitcms.org" type="hidden">
-                        <input class="search" name="q" placeholder="Search" type="text">
+                        <input type="text" name="search" placeholder="Search box" class="searchbox">
```

Comment:
This is a perfect example of what I've been talking above. The two search forms use different Google search API (form field names).